lambda/inference/index.mjs

```diff
@@ -25,34 +26,13 @@ const runChain = async ({identityId, query, model, streamingFormat}, responseStr
     const vectorStore = new LanceDB(embeddings, {table});
     const retriever = vectorStore.asRetriever();
 
-    const promptHeader = `
-    Your goal is to answer the question provided in the following question block
-    <question>
-    {question}
-    </question>
-    Important instructions:
-    You always answer the question with markdown formatting and nothing else. 
-    You will be penalized if you do not answer with markdown when it would be possible.
-    The markdown formatting you support: headings, bold, italic, links, tables, lists, code blocks, and blockquotes.
-    You do not support html in markdown. You will be penalized if you use html tags.
-    You do not support images and never include images. You will be penalized if you render images.
-    It's important you always admit if you don't know something.
-    Do not make anything up.
-    Do not answer with anything other than the markdown output.
-    `
-
-    const noContextFooter = `
-    Unfortunately no context was retrieved for this query. You should answer to the best of your capabilites,
-    but you should warn the user that no context was found. In your response, suggest the user to upload documents
-    so that you can relply with a factual answer.`;
-
-    const contextFooter = `
-    Base your answers on the context provided in the following context block.
-    Do not answer with anything other than the markdown output, 
-    and do not include the <question> or <context> within your answer and do not make up your own markdown elements. 
-    <context>
-    {context}
-    </context>`;
+    const ssmClient = new SSMClient({region:awsRegion});
+
+    const [promptHeader, noContextFooter, contextFooter] = await Promise.all([
+        getSSMParameter(ssmClient, 'promptHeader'),
```

Comment:
can you use a path like /${StackName}/${cognito_id}/promptHeader? this is a multi-tenant solution after all. My concern is that user a could change the system prompt for user b :(

Reply:
actually for the default parameter we should use `/${StackName}/default/PromptHeader` and so on :)

Reply:
https://github.com/kirtandudhatra/Serverless-Retrieval-Augmented-Generation-RAG-on-AWS/blob/dynamic-prompt-management/lambda/inference/index.mjs#L211 

That is being done inside this function.